app/controllers/application_controller.rb

```diff
@@ -32,8 +27,4 @@ def error(status_code, exception = nil)
       render status: status_code, text: error_message
     end
   end
-
-  def set_analytics_headers
-    set_slimmer_headers(format: "smart_answer")
```

Comment:
Since we are now using the meta tags component to set analytics tags `govuk:format` will change from `smart_answer` (set here) to `smartanswer_document` (the [document type of all smart answers](https://www.gov.uk/api/content/additional-commodity-code)).

To not disrupt analytics reporting we need to change the publishing-api document type into `smart_answer`.